### DISCUSSION TITLE
Glossary - Catalogue Request

### DISCUSSION THREAD

**Discussion by idolum** [eprocurementontology/eprocurementontology#496] [Ideas]
Based on the discussion in #132, I would like to propose to consider the Catalogue Request as a term of its own and something different as a catalogue. Based on the definition in given in CWA 17028-402:2016 I would like a definition like this for this term:

> A pre-award catalogue request is a catalogue of requirements on products (goods or services) intended to be bought by a contracting entity.

Mentionging "requirements" indicates that

1. no prices are given in the catalogue request
2. no specific brands of goods/services are given, but vendor-neutral specification of requirements

But these aspects might be more highlighted in a definition for this term.

**Comment by ec-mcs** [comment]
I would not limit it to goods or services. In practice you could have a catalogue on works as well. For example repairing roads, painting etc.

**Comment by idolum** [comment]
Sure! Including works is fine for me. For the sake of simplicity, I would like to have a term that can be used in definitions like this including goods, services and works. So, these three types of catalogue items have to be mentioned everytime one wants to refer to them.

**Comment by andreea-pasare** [comment]
@idolum, given the #453 issue, I see that you foresee to work on the Catalogue Response. When do you expect to work on the Catalogue Request?

**Comment by idolum** [comment]
@andreea-pasare We spent the first thoughts on it already. But mainly we would work on the catalogue request in September. As well as on the pre-award catalogue. But for both concepts, I don't expect them to be much different than the catalogue. The pre-award catalogue would be almost the same.

**Reply by andreea-pasare** [reply]
@idolum, is the new model we've been recently working on (Pre-Award Catalogue and Pre-Award Catalogue Request transaction documents) covering the needs for this issue?
